secure/time/Cargo.toml

```diff
@@ -8,3 +8,8 @@ homepage = "https://libra.org"
 license = "Apache-2.0"
 publish = false
 edition = "2018"
+
+[dependencies]
+
+libra-time = { path = "../../common/time", version = "0.1.0" }
```

Comment:
note that this is adding a dependency (albeit internal) to the TCB, but I think that's fine in this case.

Reply:
Oh, I didn't know there were rules regarding this.
Is it a problem? We can keep using SystemTime instead, it's not using it a lot.

Reply:
It's not a problem! We just have to be mindful when adding dependencies to the TCB (as you can see it updates the summaries), in this case I think the dependency is quite small and good so it's fine : )